docs/configure

```diff
@@ -714,32 +707,13 @@ virtual_domains:
                 and users.domain_id = domains.domain_id}{$value}fail}
   allow_fail
   data = ${extract{smtp}{$address_data}}
+  headers_add = X-Spam-Score: $acl_m_spam_score
+  headers_add = X-Spam-Score-Int: $spam_score_int
+  headers_add = X-Spam-Bar: $acl_m_spam_bar
```

Comment:
You have set up a forwarding address: forward@example.org -> account@example.org. For both addresses, you have enable spamassassin.

Other idea: Only add X-Spam-Flag and X-Spam-Report on real mail accounts (and not on forwarded addresses at all). Forwarded addresses are either delivered to a local mailbox (and then this setting should decide what happens), or they are delivered to an external mailserver, there we do not want this tag either (or we need to modify it).